CMakeLists.txt

```diff
@@ -89,10 +93,12 @@ target_link_libraries(threaded_io_test
                               ROOT::RIO
                               ROOT::Tree
                               TBB::tbb
+                              stdc++
```

Comment:
This shouldn't be necessary since we ask the C++ compiler to do the linking. What problems did you have?

Reply:
Same as previous comment